samples/cpp/grabcut.cpp

```diff
@@ -276,12 +276,13 @@ static void on_mouse( int event, int x, int y, int flags, void* param )
 
 int main( int argc, char** argv )
 {
-    if( argc!=2 )
+    cv::CommandLineParser parser(argc, argv, "{help h||}{@input||}");
+    if (parser.has("help"))
     {
         help();
-        return 1;
+        return 0;
     }
-    string filename = argv[1];
+    string filename = parser.get<string>("@input");
     if( filename.empty() )
     {
         cout << "\nDurn, couldn't read in " << argv[1] << endl;
```

Comment:
I think we shouldn't access `argv` in this case, it would be better to show help message.